src/main/java/com/zaxxer/hikari/pool/PoolEntry.java

```diff
@@ -70,6 +71,17 @@ final class PoolEntry implements IConcurrentBagEntry
       this.isAutoCommit = isAutoCommit;
       this.lastAccessed = currentTime();
       this.openStatements = new FastList<>(Statement.class, 16);
+      boolean isJDBC43OrLater = false;
```

Comment:
any specific reason to use `isJDBC43OrLater` inside the static initializer block? Could this be moved?